Packs/Threat_Crowd/Integrations/ThreatCrowd_v2/ThreatCrowd_v2.py

```diff
@@ -0,0 +1,301 @@
+        outputs = {
+            'hashes': hashes,
+            'permalink': res.get('permalink'),
+            'resolutions': resolutions,
+            'references': res.get('references'),
+            'response_code': res.get('response_code'),
+            'votes': res.get('votes'),
+            'value': res.get('value')
+        }
+        command_results.append(CommandResults(
+            outputs_prefix='ThreatCrowd.IP',
+            outputs=outputs,
+            outputs_key_field='value',
+            indicator=ip_object,
+            readable_output=markdown,
+            raw_response=res
+
+        ))
+
+    return command_results
+
+
+def email_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    command_results: List[CommandResults] = []
+    api_url = 'email/report/'
+    emails = argToList(args.get('email'))
+    for email in emails:
+        res = client._http_request(method='GET', url_suffix=api_url, params={'email': email})
+        res['value'] = email
+        score, score_str = _get_dbot_score(res)
+        dbot = Common.DBotScore(
+            email, DBotScoreType.EMAIL, VENDOR, score, reliability=client.reliability)
+        email_object = Common.EMAIL(email, dbot)
+
+        markdown = f"Threat crowd report for Email {email} \n"
+        markdown += f'DBotScore: {score_str} \n'
+        markdown += tableToMarkdown("Results", res)
+
+        command_results.append(CommandResults(
+            outputs_prefix='ThreatCrowd.Account',
+            outputs=res.copy(),
+            outputs_key_field='value',
+            indicator=email_object,
+            readable_output=markdown,
+            raw_response=res
+
+        ))
+
+    return command_results
+
+
+def domain_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    command_results: List[CommandResults] = []
+    api_url = 'domain/report/'
+    resolution_limit = None
+    domains = argToList(args.get('domain'))
+    for domain in domains:
+        res = client._http_request(method='GET', url_suffix=api_url, params={'domain': domain})
+        res['value'] = domain
+        score, score_str = _get_dbot_score(res)
+        dbot = Common.DBotScore(
+            domain, DBotScoreType.DOMAIN, VENDOR, score, reliability=client.reliability)
+        domain_object = Common.Domain(domain, dbot)
+
+        markdown = f"### Threat crowd report for domain {domain} \n"
+        markdown += f'DBotScore: {score_str} \n'
+
+        if not client.extended_data:
+            resolution_limit = DEFAULT_RESOLUTION_LIMIT
+
+        subdomains = res.get('subdomains')[:resolution_limit]
+        resolutions = handle_resolutions(res.get('resolutions', []), resolution_limit)
+        markdown += tableToMarkdown('Resolutions', resolutions)
+        res_without_resolutions = res.copy()
+        res_without_resolutions.pop('resolutions')
+        markdown += tableToMarkdown("\n", res_without_resolutions)
```

Comment:
let's look at it in the demo, I think it will be nicer to show the md without the resolutions first, and only then the resolutions.